js/sum_mixed_array.js

```diff
@@ -0,0 +1,4 @@
+function sumMix(x){
+  return x.reduce((a, b) => Number(a) + Number(b), 0);
```

Comment:
As a side exercise, find out the difference between `parseInt` and `Number` 😄  Just for your information